packages/main/src/SegmentedButton.ts

```diff
@@ -96,10 +52,60 @@ const metadata = {
  * @appenddocs sap.ui.webc.main.SegmentedButtonItem
  * @public
  */
+
+@customElement("ui5-segmented-button")
+@languageAware
+/**
+ * Fired when the selected item changes.
+ *
+ * @event sap.ui.webc.main.SegmentedButton#selection-change
+ * @param {HTMLElement} selectedItem the pressed item.
+ * @public
+ */
+@event("selection-change", {
+	detail: {
+		selectedItem: { type: HTMLElement },
+	},
+})
 class SegmentedButton extends UI5Element {
-	static get metadata() {
-		return metadata;
-	}
+	/**
+	 * Defines the accessible ARIA name of the component.
+	 *
+	 * @type {string}
+	 * @defaultvalue: ""
+	 * @public
+	 * @name sap.ui.webc.main.SegmentedButton.prototype.accessibleName
+	 * @since 1.0.3
+	 */
+	@property({ defaultValue: undefined })
+	accessibleName?: string;
+
+	/**
+	 * Defines the items of <code>ui5-segmented-button</code>.
+	 * <br><br>
+	 * <b>Note:</b> Multiple items are allowed.
+	 * <br><br>
+	 * <b>Note:</b> Use the <code>ui5-segmented-button-item</code> for the intended design.
+	 * @type {sap.ui.webc.main.ISegmentedButtonItem[]}
+	 * @name sap.ui.webc.main.SegmentedButton.prototype.default
+	 * @slot items
+	 * @public
+	 */
+	@slot({ type: HTMLElement, "default": true })
+	items!: Array<SegmentedButtonItem>;
+
+	static i18nBundle: I18nBundle;
+
+	_itemNavigation: ItemNavigation;
+
+	absoluteWidthSet: boolean // set to true whenever we set absolute width to the component
+	percentageWidthSet: boolean; //  set to true whenever we set 100% width to the component
```

Comment:
Is the comment really needed here? If so, remove the double space before set

Reply:
Same as above, I also removed the space